### PR TITLE
Allow to configure CORS exposed headers

### PR DESCRIPTION
**Pull request by apuig**

### CODE REVIEW

**Comment by sergicastro** [comment]
:+1:

File: src/main/java/com/abiquo/commons/web/filter/BaseCORSFilter.java [hunk 1]
```diff
@@ -92,6 +102,22 @@ public void doFilter(final ServletRequest request, final ServletResponse respons
 
                 res.addHeader(ACCESS_HEADERS_CORS_HEADER, sb.toString());
             }
+
+            List<String> exposedHeaders = exposedHeaders();
+            if (exposedHeaders != null && !exposedHeaders.isEmpty())
+            {
+                StringBuilder sb = new StringBuilder();
```
Comment (danielestevez) [inline src/main/java/com/abiquo/commons/web/filter/BaseCORSFilter.java]: StringUtils.join(exposedHeaders " ,") 
:smiling_imp: Don't tell me you didn't expect it

Comment (apuig) [inline src/main/java/com/abiquo/commons/web/filter/BaseCORSFilter.java]: StringJoiner [https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html] works for you ?

Comment (danielestevez) [inline src/main/java/com/abiquo/commons/web/filter/BaseCORSFilter.java]: I can live with it

**Comment by danielestevez** [comment]
:+1: awesome codestyle

**Pull request merged**